rllib/algorithms/bc/torch/bc_torch_learner.py

```diff
@@ -49,11 +50,13 @@ def possibly_masked_mean(t):
             mask = None
             possibly_masked_mean = torch.mean
 
-        action_dist_class_train = self.module[module_id].get_train_action_dist_cls()
+        action_dist_class_train = (
+            self.module[module_id].unwrapped().get_train_action_dist_cls()
```

Comment:
Not for this PR, but I wonder whether we should make the MARLModule return already the unwrapped RLModule automatically when we access a sub-module through `__getitem__()`. ...

Reply:
Do we need anywhere the wrapped module? I guess not even in the `DDP` case.

Reply:
Ah, hold on, good point. The most user-friendly way is probably to make sure the wrapper exposes:
* all RLModule base methods.
* all API methods that the wrapped module implements -> So the wrapper should check, what RLModule APIs (ValueFunctionAPI, TargetNetworkAPI, etc..) its wrapped module implements, then expose all this API's methods as well automatically. ?

Reply:
This is how it could work. Its not a very elegant design but I tried around some weeks ago to make it elegant by just defining all methods not yet published from the wrapped module and this is non-trivial. Have to check agaon what the reasons were.